backend/src/channel/exceptions/channel.exception.ts

```diff
@@ -6,6 +6,13 @@ export class ExceptionTryingToUpdateChannelID extends HttpException {
   }
 }
 
+export class ExceptionTryingToUpdateOwnerID extends HttpException {
```

Comment:
utile ?

Reply:
If you try to update owner ID in the Update Method and not in updateOwnerMethod